tests/commons/j2/test_j2_functions.py

```
@@ -139,6 +139,42 @@ def test_datetime(self):
                     xsd_fmt(v, type_name), fmt, "bad %s format" % type_name
                 )
 
+    def test_gyear(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_gyear`

Add return type annotation: `None`

(ANN201)